packages/jest-jasmine2/src/__tests__/iterators-test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+'use strict';
+
+describe('compare iterables correctly', () => {
+
+  it('works for arrays', () => {
+    const obj = [1, {}, []];
```

Comment:
is it an array or an object?

I know this might sound silly, but in Jest the code style is not to abbreviate words. So this should be called `array` (or well, `object` but that doesn't go well with the test).